### PR TITLE
Update AWS TypeScript handler template

### PR DESCRIPTION
**Pull request by uneco**
## What did you implement:

`aws-nodejs-typescript` handler template of new notation similar to:
https://github.com/serverless/serverless/blob/master/lib/plugins/create/templates/aws-nodejs/handler.js

## How did you implement it:

Just like the js version

## How can we verify it:

`sls create --template aws-nodejs-typescript`

## Todos:

- [x] Enable "Allow edits from maintainers" for this PR
- [x] Update the messages below

***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO

### CODE REVIEW

File: lib/plugins/create/templates/aws-nodejs-typescript/handler.ts [hunk 1]
```diff
@@ -1,13 +1,11 @@
-import { APIGatewayEvent, Callback, Context, Handler } from 'aws-lambda';
+import { APIGatewayProxyHandler } from 'aws-lambda';
 
-export const hello: Handler = (event: APIGatewayEvent, context: Context, cb: Callback) => {
-  const response = {
+export const hello: APIGatewayProxyHandler = async (event, context) => {
```
Comment (franciscocpg) [inline lib/plugins/create/templates/aws-nodejs-typescript/handler.ts]: Why did you remove the types from the function parameters?

Comment (uneco) [reply lib/plugins/create/templates/aws-nodejs-typescript/handler.ts]: in the current template, it was necessary to explicitly indicate the type of the arguments -- that's because the used insufficient type `Handler`.
now, by storing function to the constant of type `APIGatewayProxyHandler`, the types of `event` and `context` are implicitly inferred. Please see the attached image.

screenshot

Comment (franciscocpg) [reply lib/plugins/create/templates/aws-nodejs-typescript/handler.ts]: hi @uneco 
Thanks for the explanation.

Just saw https://github.com/DefinitelyTyped/DefinitelyTyped/blob/b697ec8a1bb26313b4d56a0af66b9c45c9920cef/types/aws-lambda/index.d.ts#L814 after your explanation and I understood.

:+1:
